packages/base-styles/_mixins.scss

```diff
@@ -1,6 +1,74 @@
 @import "./functions";
 @import "./long-content-fade";
 
+/**
+ * Typography
+ */
+
+@mixin text-heading() {
+	font-family: $font-family-headings;
+	font-weight: $font-weight-medium;
+}
+
+@mixin text-body() {
+	font-family: $font-family-body;
+	font-weight: $font-weight-regular;
+}
```

Comment:
Since these mixins are (I believe) meant to be only used in this file (at least for now), we could prefix their name with a `_` to better indicate their "private" nature — ie. `_text-heading` and `_text-body`